pyiron_base/generic/filedata.py

```diff
@@ -0,0 +1,177 @@
+import json
+import os
+import pandas
+
+from pyiron_base import ImportAlarm, FileHDFio
+
+__author__ = "Niklas Siemer"
+__copyright__ = (
+    "Copyright 2020, Max-Planck-Institut für Eisenforschung GmbH - "
+    "Computational Materials Design (CM) Department"
+)
+__version__ = "0.1"
+__maintainer__ = "Niklas Siemer"
+__email__ = "siemer@mpie.de"
+__status__ = "development"
+__date__ = "Feb 02, 2021"
+
+
+_has_imported = {}
+import_alarm = ImportAlarm()
+_not_imported = ''
+try:
+    from PIL import Image
```

Comment:
Should we add `pillow` to the requirements?

Reply:
I don't know. For a the display of display and so on this is of course very nice. However, since we 'just' convert a image file to an image object and return this object and don't (yet) provide functionality to _do_ anything with this object I was trying to keep our dependency list shorter.